### PR TITLE
Ignore libcrypto.ld and libssl.ld

### PR DESCRIPTION
**Pull request by mattcaswell**
These are auto generated files that should not be checked into git

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -84,6 +84,8 @@ Makefile
 /TAGS
 /libcrypto.map
 /libssl.map
+/libcrypto.ld
+/libssl.ld
```
Comment (levitte) [inline .gitignore]: I would make that `*.ld`, and while we're at it, we should probably also ignore `*.map`

Comment (mattcaswell) [reply .gitignore]: Done

**Comment by mattcaswell** [comment]
Fixup commit pushed.

File: .gitignore (hunk 1 above)
Comment (levitte) [reply .gitignore]: For future reference, not just in root.

Comment (mattcaswell) [reply .gitignore]: Are you asking me to change .gitignore, to ignore those files anywhere? .ld files and .map files? Why might they appear anywhere?

Comment (levitte) [reply .gitignore]: Yes.
Think future modules that we want to make similar ld scripts for. Say we'd like to do that for engines (we really should). Where are those ld scripts going to end up?

Comment (mattcaswell) [reply .gitignore]: Updated again

**Comment by mattcaswell** [comment]
Pushed. Thanks.

**Pull request closed without merging**